src/main/java/com/teamfive/disscard/dto/Card.java

```diff
@@ -39,7 +39,10 @@
     @SerializedName("popularity")
     public int popularity;
 
-    public String ToString() {
-        return cardName + " (" + series + ")";
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(cardName).append(" (").append(series).append(")");
+        return sb.toString();
```

Comment:
Added a string builder to toString to improve efficiency of the toString method to a method that is called often, which minimizes the overhead. 
https://docs.oracle.com/javase/8/docs/api/java/lang/StringBuilder.html